common/scala/src/main/scala/whisk/core/entity/AuthKey.scala

```diff
@@ -16,6 +16,13 @@
 
 package whisk.core.entity
 
+import scala.util.Try
+
+import spray.json.JsString
+import spray.json.JsValue
+import spray.json.RootJsonFormat
+import spray.json.deserializationError
```

Comment:
> But I'ld strongly recommend to configure your IDE to leave Wildcard imports alone

We need to automate and standardize such conventions otherwise...